From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Fri, 10 Apr 2020 16:16:08 +0800
Subject: media: gpu: v4l2: Use POLLIN for pending event

The v4l-rkmpp is using eventfd to fake poll events which not supporting
POLLPRI.

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>

diff --git a/media/gpu/v4l2/v4l2_device.cc b/media/gpu/v4l2/v4l2_device.cc
index 5ec769913b0c316abd0526300a0227182e4cd09f..17704421f53e758b937cd2691e15d6d346d2b8a9 100644
--- a/media/gpu/v4l2/v4l2_device.cc
+++ b/media/gpu/v4l2/v4l2_device.cc
@@ -363,7 +363,11 @@ bool V4L2Device::Poll(bool poll_device, bool* event_pending) {
     VPLOGF(1) << "poll() failed";
     return false;
   }
-  *event_pending = (pollfd != -1 && pollfds[pollfd].revents & POLLPRI);
+
+  // HACK: Could not fake POLLPRI with eventfd
+  // *event_pending = (pollfd != -1 && pollfds[pollfd].revents & POLLPRI);
+  *event_pending = (pollfd != -1 && pollfds[pollfd].revents & POLLIN);
+
   return true;
 }
 
